pkg/pods/pod_template.go

```diff
@@ -18,12 +18,12 @@ import (
 	"net/http"
 
 	"github.com/tektoncd/cli/pkg/file"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/pod"
 	"sigs.k8s.io/yaml"
 )
 
-func ParsePodTemplate(httpClient http.Client, podTemplateLocation string, validate file.TypeValidator, errorMsg error) (v1beta1.PodTemplate, error) {
-	podTemplate := v1beta1.PodTemplate{}
+func ParsePodTemplate(httpClient http.Client, podTemplateLocation string, validate file.TypeValidator, errorMsg error) (pod.PodTemplate, error) {
+	podTemplate := pod.PodTemplate{}
```

Comment:
Arf.. it should have been `pod.Template`.. but oh well.

Reply:
Oh actually that's the case already, `PodTemplate` is an alias to `Template`. Ideally we would use `Template` there.

Reply:
they have both :D